src/go/rpk/pkg/cli/cmd/start.go

```diff
@@ -523,6 +548,67 @@ func parseFlags(flags []string) map[string]string {
 	return parsed
 }
 
+func parseSeeds(seeds []string) ([]config.SeedServer, error) {
```

Comment:
I could be reading this wrong, but it seems you must specify the seed servers as "host:port+id," it would be great if we had some flexibility, that if you use "host," we use the default port and that id is also optional.

Reply:
`parseSeeds` could probably do with some tests.

Reply:
@BenPope It could. I will add them.

Reply:
@rkruze the ID definitely needs to go there, because redpanda requires it.
The port could be optional, however.

Reply:
https://github.com/vectorizedio/redpanda/issues/190